compiler/lib/src/main/scala/codegen/CppWriter/StringCppWriter.scala

```diff
@@ -76,9 +76,14 @@ case class StringCppWriter(
         CppDoc.Lines(
           List(
             CppDocHppWriter.writeAccessTag("public"),
-            addBlankPrefix(wrapInEnum(lines(
-              s"SERIALIZED_SIZE = $size + sizeof(FwBuffSizeType) //!< Size of buffer + storage of two size words"
-            ))),
+            addBlankPrefix(
+              wrapInEnum(
+                List(
+                  line("//!< The size of the string length plus the size of the string buffer"),
```

Comment:
I think the `<` shouldn't be there since the comment refers to the line below it.

Reply:
Good catch, fixed.